characterisation/triggers/TRPS0011.test.ts

```diff
@@ -0,0 +1,57 @@
+import World from "../../utils/world"
+import generatePhase2Message from "../helpers/generatePhase2Message"
+import { processPhase2Message } from "../helpers/processMessage"
+import MessageType from "../types/MessageType"
+import { TriggerCode } from "../types/TriggerCode"
+
+const code = TriggerCode.TRPS0011
+
+describe.ifPhase2("TRPS0011", () => {
+  afterAll(async () => {
+    await new World({}).db.closeConnection()
+  })
+
+  it("creates a TRPS0011 for AnnotatedHearingOutcome when no operations and exceptions are generated", async () => {
+    const inputMessage = generatePhase2Message({
+      messageType: MessageType.ANNOTATED_HEARING_OUTCOME,
+      hoTemplate: "NoOperationsAndExceptions"
+    })
+
+    const { triggers } = await processPhase2Message(inputMessage, { recordable: false })
```

Comment:
Do we need to set `recordable: false` for all of these? It looks like `recordable` is only used for Phase 1 for mocking the PNC.